doc/source/wsrep-files-index.rst

```diff
@@ -29,7 +29,7 @@
       /*!50003 SET @OLD_COMPLETION_TYPE=@@COMPLETION_TYPE,COMPLETION_TYPE=0*/;
       DELIMITER /*!*/;
       # at 4
-      #160809  16:04:05 server id 3  end_log_pos 123     Start: binlog v 4, server v 5.7.12-5rc1-log created 160809 16:04:05 at startup
+      #160809  16:04:05 server id 3  end_log_pos 123     Start: binlog v 4, server v 8.0-log created 160809 16:04:05 at startup
```

Comment:
Use the example form the latest version instead of just incorrectly changing the version

Reply:
is the suggested form of analysis used at all? a combination of head and cat (as provided in the code block for GRA_*.log entry) are low level and it feels like there should be a better way. i would suggest removing the code block entirely.